apps/guardian/src-tauri/src/main.rs

```diff
@@ -1122,12 +1540,8 @@ async fn connect(app_config_state: State<'_, AppConfigState>) -> Result<Election
 /// ## Errors
 /// - [`Error::NodeConnection`]
 /// - [`Error::NetworkError`]
-async fn energy_to_ccd<'a>(
-    energy: Energy,
-    app_config_state: &State<'a, AppConfigState>,
-) -> Result<Amount, Error> {
-    let mut app_config_guard = app_config_state.0.lock().await;
-    let mut node = app_config_guard.connection().await?.contract.client;
+async fn energy_to_ccd(energy: Energy, app_config: &mut AppConfig) -> Result<Amount, Error> {
```

Comment:
I'm guessing you realize this, but it's because I need a mutable node client to get the chain parameters to do the calculation.

In any case where this is used, I'll have the client available so I'll change the signature to take a client instead.